Podcast/BookmarkViewController.swift

```diff
@@ -44,8 +45,16 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        bookmarkTableView.reloadData()
         
+        // check before reloading data whether the Player has stopped playing the currentlyPlayingIndexPath
+        if let indexPath = currentlyPlayingIndexPath {
+            if let card = cards[indexPath.row] as? EpisodeCard {
+                if Player.sharedInstance.currentEpisode?.id != card.episode.id {
```

Comment:
How should I do this? I think I have to unwrap the indexPath before I use it to get the correct card from cards